scripts/ci/test-examples.sh

```diff
@@ -41,17 +43,25 @@ build_local
 
 # Run Geant4 app examples
 if [ -z "${CELER_DISABLE_ACCEL_EXAMPLES}" ]; then
+  if [ -z "${G4VERSION_NUMBER}" ]; then
+    # Get the geant4 version 11.2.3, replace `.` with ` `, concat as 110203
+    _vers=$(geant4-config --version | tr '.' ' ')
+    G4VERSION_NUMBER=$(printf '%d%02d%02d' ${_vers})
```

Comment:
I don't know that it matters but the `G4Version.h` claims that the 2nd and 3rd numbers are restricted to a single digit and write the version down as `1123`. I.e.:
```suggestion
    # Get the geant4 version 11.2.3, replace `.` with ` `, concat as 110203
    _vers=$(geant4-config --version | tr '.' ' ')
    G4VERSION_NUMBER=$(printf '%d%01d%01d' ${_vers})
```